channels/api.py

```diff
@@ -800,9 +828,19 @@ def create_comment(self, text, post_id=None, comment_id=None):
             raise ValueError("Exactly one of post_id and comment_id must be provided")
 
         if post_id is not None:
-            return self.get_post(post_id).reply(text)
+            comment = self.get_post(post_id).reply(text)
         else:
-            return self.get_comment(comment_id).reply(text)
+            comment = self.get_comment(comment_id).reply(text)
+
+        _, link_id = get_kind_and_id(comment.link_id)
+
+        Comment.objects.create(
+            comment_id=comment.id,
+            post=Post.objects.get(post_id=link_id),
```

Comment:
Couldn't this just be `post_id=link_id,`?

Reply:
`link_id` here is the reddit link id, not the postgres primary key, so the relation here is `post__post_id`.